templates/seL4AllocatorMempool.template.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2022, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <camkes.h>
+
+/*- set mempool_size = macros.ROUND_UP(configuration[me.name].get('allocator_mempool_size', 83886080), macros.PAGE_SIZE) -*/
```

Comment:
can you make this `80*1024*1024` to make it a bit nicer?

Reply:
Sure